templates/es6/routes/index.js

```diff
@@ -0,0 +1,9 @@
+import express from 'express';
```

Comment:
I think it's better to leave the express object still accessible, since people are going to be building other projects off of this. Don't want to be too concise :)